### DISCUSSION TITLE
Is it possible to call appium mobile automation tests from inside the cypress framework or vice versa?

### DISCUSSION THREAD

**Discussion by bimanpathirage** [cypress-io/cypress#27752] [Questions and Help]
I have a use case where there is a user who has to log in to a web app and then to a mobile app and do some tests, and then login again to the web app and verify that mobile activity within the same test flow.

For such a scenario, can I somehow call the appium tests from inside the cypress automation code that I have written for the web app?

Or is it advisable to write the web app automation in selenium rather than cypress?

Currently I have 2 separate test frameworks for web app (cypress, js) and mobile app (appium, .net). I'm trying to run these tests in one flow. I'm new to using appium for mobile automation, so any help regarding this is highly appreciated. Thanks in advance.

**Comment by AwakeSnake** [comment]
Did you find a solution to this? I also have an appium test suite that i'd like to incorporate pre-existing cypress tests into

**Reply by bimanpathirage** [reply]
Nope. I moved to Selenium from Cypress for this project 😥
